common/tests/sm_int/sm_int.c

```diff
@@ -1635,11 +1637,57 @@ TEST_FUNCTION(sm_close_begin_with_cb_triggers_cancel)
     ThreadAPI_Sleep(500);
 
     /// assert
-    ASSERT_ARE_EQUAL(SM_RESULT, sm_close_begin_with_cb(context.sm, cancellingCallback, (void*)(&context.cancel_wait)), SM_EXEC_GRANTED);
+    ASSERT_ARE_EQUAL(SM_RESULT, sm_close_begin_with_cb(context.sm, cancellingCallback, (void*)(&context.cancel_wait), NULL, NULL), SM_EXEC_GRANTED);
```

Comment:
>  ASSERT_ARE_EQUAL(SM_RESULT, sm_close_begin_with_cb(context.sm, cancellingCallback, (void*)(&context.cancel_wait), NULL, NULL), SM_EXEC_GRANTED); [](http://example.com/codeflow?start=2&length=146)

here and below, should be ASSERT_ARE_EQUAL(type, EXPECTED, ACTUAL) so that the error message makes sense (so the args are flipped right now)